src/app.module.ts

```diff
@@ -15,14 +18,26 @@ import { AppService } from './app.service';
         port: +configService.get('POSTGRES_PORT'),
         username: configService.get('POSTGRES_USER'),
         password: configService.get('POSTGRES_PASSWORD'),
-        database: configService.get('POSTGRES_DB'),
+        database:
+          process.env.NODE_ENV !== 'test'
+            ? configService.get('POSTGRES_DB')
+            : 'auth_test_db',
         autoLoadEntities: true,
         synchronize: true,
```

Comment:
`synchronize: true` should be used only in development/test environment.

Reply:
Resolved on 11e4cc9e471ea4106280df108adbaef7e46426c9.